CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# INDIGO IAM developer guide
+
+Thank you for investing your time in contributing to our project!
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+## Development environment
+
+The INDIGO IAM service is a [Maven][maven] project build with Java 17.  
+Run
+
+```
+$ mvn package
+```
+
+to build the project, or
+
+```
+$ mvn package -DskipTests
+```
+to skip tests execution.
+
+You can use your favorite IDE for development (Eclipse is the one adopted at the time of writing).
+Install the `Spring Tools 4` plugin to use Spring buttons and configurations. Please use a modified Google style
```

Comment:
In case you're using Eclipse, etc..
Visual Studio Code has a similar extension that needs to be installed